src/systemTest/java/com/flightstats/hub/callback/CacheObject.java

```diff
@@ -0,0 +1,33 @@
+package com.flightstats.hub.callback;
+
+import com.flightstats.hub.callback.model.RequestObject;
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+@Slf4j
+public class CacheObject {
+
+    private static final Map<String, List<String>> cacheRequestObject = new ConcurrentHashMap<>();
+
+    public void put(RequestObject requestObject) {
+        String webhookName = requestObject.getName();
+
+        synchronized (CacheObject.class) {
```

Comment:
You've got a ConcurrentHashMap, but then you're using it in a synchronized block which sort of defeats the value of a ConcurrentHashMap.

Reply:
synchronized is for code checking the map. That is not synchronized.